TeamPage.html

```diff
@@ -947,14 +933,14 @@ <h2>Meet Our Team</h2><br>
                   <div class="modal-content">
                     <div class="modal-header">
                       <div class="modal-title">
-                        <p class="text-justify"><b>Natalie Renkes</b> - Project Manager<br>
+                        <p class="text-justify"><b>Natalie Renkes</b> - Junior Project Manager<br>
                           Work Location: Remote<br>
                           Email: natalie.renkes@unlv.edu</p>
                       </div>
                       <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
                     </div>
                     <div class="modal-body">
-                      <p class="text-justify">Natalie Renkes joined the UNLV-FIA group in May 2020. She anticipates receiving her M.S. degree in Geoscience from UNLV in the spring of 2023. She supervises student interns/staff members, is project manager for the DATIM, EVALIDator, and Sample Nonresponse Bias projects, and performs other duties based on FIA National Operations priorities. As a member of the FIA Section 508 Program Development Team, she is also working to ensure our group's compliance with federal accessibility laws. Outside of UNLV-FIA, Natalie’s research interests focus on naturally-occurring asbestos and its health implications within metropolitan Las Vegas. In her spare time, Natalie enjoys playing The Legend of Zelda: Breath of the Wild, hiking/camping, and is an avid reader. Her favorite book series is Game of Thrones (she hasn’t seen the show, don’t ruin the ending)!
+                      <p class="text-justify">Natalie Renkes joined the UNLV-FIA group in May 2020. She received her M.S. degree in Geoscience from UNLV in May of 2023. She supervises student interns/staff members, is project manager for the DATIM, EVALIDator, Sample Nonresponse Bias in Estimations, and Science Strategy projects; and performs other duties based on FIA National Operations priorities. As a member of the FIA Section 508 Program Development Team, she is also working to ensure our group's compliance with federal accessibility laws. Outside of UNLV-FIA, Natalie’s research interests focus on naturally-occurring asbestos and its health implications within metropolitan Las Vegas. In her spare time, Natalie enjoys playing The Legend of Zelda: Breath of the Wild, hiking/camping, and is an avid reader. Her favorite book series is Game of Thrones (she hasn’t seen the show, don’t ruin the ending).

```

Comment:
Just to match Natalie's exact wording, I think this bio should end with an exclamation mark. But if you disagree, let me know why and we can just ignore this!